packages/docs/components/code-sandbox.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <!-- 'code-options' is a build-in prop, do not edit it -->
+  <Sandbox
+    :rtl="rtl"
+    :template="template"
+    :light-theme="lightTheme"
+    :dark-theme="darkTheme"
+    :options="{
+      ...props, // do not forget it
+      coderHeight: Number(props.coderHeight),
+      previewHeight: Number(props.previewHeight),
+      showLineNumbers: true,
+    }"
+    :custom-setup="{
+      ...props, // do not forget it
+      deps: {
+        yjs: 'latest',
+        '@toeverything/theme': 'latest',
```

Comment:
Just curious if we can use the `nightly` version here?

Reply:
ok, you can update it.